pandas/io/excel/_base.py

```diff
@@ -1629,6 +1629,29 @@ def book(self):
 
     @property
     def sheet_names(self):
+        """
+        Names of the sheets in the document.
+
+        This is mainly useful for loading a specific sheet into a DataFrame when
```

Comment:
nit: I'm not sure the "mainly useful" part is true. Recommend `particularly` instead of `mainly`.